index.html

```diff
@@ -73,253 +79,420 @@ <h1 class="page-background" id="principal-title" class="color-blue fonts">I’m
+      </div>
+    </div>
+
+    <div class="grid-container card1 grid-container-desktop">
+      <div class="card-container card-container-desktop">
+        <div class="container2-desktop">
+          <div class="card-desktop">
+            <div class="card-img2">
+              <img class="card-img" src="./vectors/Snapshoot Portfolio4.png" alt="image of my project" />
+            </div>
+            <div>
+              <div class="text-container">
+                <h2 class="card-title">4- Multi-Post
+                  Stories</h3>
+                  <div class="frame-2">
+                    <p class="client">CANOPY</p>
+                    <img src="./vectors/Counter.png" alt="Counter">
+                    <p class="Role">Back End Dev</p>
+                    <img src="./vectors/Counter.png" alt="Counter">
+                    <p class="Role">2023</p>
+                  </div>
+                  <p class="card-description">
+                    A daily selection of privately personalized reads; no accounts or sign-ups required.
+                  </p>
+
+
+                  <ul class="tags-list-style-none">
+                    <li class="tag-description1">HTML</li>
+                    <li class="tag-description2">CSS</li>
+                    <li class="tag-description3">JAVASCRIPT</li>
+                  </ul>
+                  <button type="button" class="see-btn">See project</button>
+              </div>
+            </div>
+
+          </div>
+        </div>
+
 
+
+      </div>
     </div>
+    <div class="grid-container2  grid-container-desktop">
+      <div class="card-container card-container-desktop">
+        <div class="container2-desktop">
+          <div class="card-desktop card-container-reverse">
+            <div class="card-img2">
+              <img class="card-img5" src="./vectors/Snapshoot Portfolio8.png" alt="image of my project" />
+            </div>
 
-    <ul class="header2">
-      <li>
-        <p class="languages">Skills</p>
-      </li>
+            <div class="text-container">
+              <h2 class="card-title">4-Uber Navigation</h3>
+                <div class="frame-2">
+                  <p class="client">CANOPY</p>
+                  <img src="./vectors/Counter.png" alt="Counter">
+                  <p class="Role">Back End Dev</p>
+                  <img src="./vectors/Counter.png" alt="Counter">
+                  <p class="Role">2023</p>
+                </div>
+                <p class="card-description">
+                  A daily selection of privately personalized reads; no accounts or sign-ups required.
+                </p>
+
+
+                <ul class="tags-list-style-none">
+                  <li class="tag-description1">HTML</li>
+                  <li class="tag-description2">CSS</li>
+                  <li class="tag-description3">JAVASCRIPT</li>
+                </ul>
+                <button type="button" class="see-btn">See project</button>
+            </div>
```

Comment:
- [ ] **[OPTIONAL]** Please make sure your work section card design stays aligned with the figma design even when the screen is resized. This is a requirement missed by the previous reviewer so please consider implementing it.
**When your card design is resized**
![image](https://user-images.githubusercontent.com/74411135/229291145-6ac18247-5ef6-4a9d-9576-8cf79c962740.png)

This will make your project way more prettier which will improve the user experience 📱  😄 👍